src/red.ts

```diff
@@ -213,13 +214,16 @@ export const serializedRedEnvSourceText = (function redEnvFactory(blueEnv: Membr
     }
 
     function copyRedOwnDescriptors(shadowTarget: RedShadowTarget, blueDescriptors: PropertyDescriptorMap) {
-        for (const key in blueDescriptors) {
+        // using ownKeys here to get all string/symbol/number property names
+        const keys = ownKeys(blueDescriptors);
+
+        ArrayForEach(keys, (key: string | symbol | number) => {
             // avoid poisoning by checking own properties from descriptors
             if (hasOwnPropertyCall(blueDescriptors, key)) {
-                const originalDescriptor = getRedDescriptor(blueDescriptors[key]);
+                const originalDescriptor = getRedDescriptor(ReflectGet(blueDescriptors, key));
```

Comment:
Why `ReflectGet` here?

Reply:
`PropertyDescriptorMap` in typescript does not allow Symbols to be used as keys and the index notation does not pass the compiler. Using ReflectGet bypasses that. I don't know how to make this work better.

Reply:
I think we can do a TS comment to ignore it.

Reply:
that's better, thanks!